traffic_control/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/kubedl/istio_less_ingress_controller
+
+go 1.15
+
+require (
+	github.com/caddyserver/caddy/v2 v2.4.5
+	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac // indirect
+	k8s.io/api v0.20.7
+	k8s.io/apimachinery v0.20.7
+	k8s.io/client-go v0.20.7
+	k8s.io/klog v1.0.0
+	k8s.io/utils v0.0.0-20210820185131-d34e5cb4466e // indirect
```

Comment:
we shouldn't have another go.mod file in a sub package ?

Reply:
Actually I think it better to be placed under a separate repo. but anyway if it is placed under the kubedl, l think it's ok because this service is more like a plugin, it should not pollute the main go.mod